bread/tests/settings.py

```diff
@@ -1,6 +1,10 @@
 from bread.settings.required import *  # noqa
 
-INSTALLED_APPS = BREAD_DEPENDENCIES + ["django.contrib.admin"]  # noqa
+INSTALLED_APPS = BREAD_DEPENDENCIES + [  # noqa
+    "django.contrib.admin",
+    "bread.contrib.reports",
+    "bread.contrib.triggers",
+]
```

Comment:
Should be here anyway, so the contribs are covered when running tests or doing migrations.